src/main/resources/application.yml

```diff
@@ -15,9 +15,6 @@ spring:
 
   jpa:
     open-in-view: false
-    properties:
```

Comment:
Spring reported this as not needed on application started as it can determine the DB dialect itself.